chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/statistics/SummaryStatisticsDAOImpl.java

```diff
@@ -0,0 +1,61 @@
+package gov.healthit.chpl.dao.statistics;
+
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import gov.healthit.chpl.auth.Util;
+import gov.healthit.chpl.dao.impl.BaseDAOImpl;
+import gov.healthit.chpl.entity.SummaryStatisticsEntity;
+import gov.healthit.chpl.exception.EntityCreationException;
+import gov.healthit.chpl.exception.EntityRetrievalException;
+
+@Repository("summaryStatisticsDAO")
+public class SummaryStatisticsDAOImpl extends BaseDAOImpl implements SummaryStatisticsDAO {
+    private static final long MODIFIED_USER_ID = -3L;
```

Comment:
There are a number of DAO classes that have the same MODIFIED_USER_ID defined. Could you move that into some common util-type place as a public static variable?